Sources/Datadog/Core/FeaturesConfiguration.swift

```diff
@@ -150,6 +149,9 @@ extension FeaturesConfiguration {
             rumEndpoint = .custom(url: customRUMEndpoint.absoluteString)
         }
 
+        let source = (configuration.additionalConfiguration[CrossPlatformAttributes.ddsource] as? String) ?? Datadog.Constants.ddsource
+        let sdkVersion = (configuration.additionalConfiguration[CrossPlatformAttributes.sdkVersion] as? String) ?? __sdkVersion
```

Comment:
maybe we can use both cross platform SDK version and native SDK version as `sdkVersion` in order not to lose version information, wdyt?

for example if RN passes `RN-1.2.3` to native SDK version `v4.5.6` we can merge them into one somehow: like `RN-1.2.3/iOS-4.5.6`?

Reply:
Could be confusing for the user, no? 
Maybe it would be best to remove the `~>` from [here](https://github.com/DataDog/dd-bridge-ios/blob/main/DatadogSDKBridge.podspec#L20) and [here](https://github.com/DataDog/dd-sdk-reactnative/blob/main/packages/core/DatadogSDKReactNative.podspec#L20) so we know with certainty which version `RN-1.2.3` depends on.

Reply:
Interesting point 👍. I think that knowing the version of underlying native SDK might be important for us in troubleshooting, but not especially for our users.

As we leverage `~> x.y.z` for version pinning, it means that given version of RN SDK will use `x.y.*` version of the native SDK - so _major_ and _minor_ are pinned, and only _patch_ component is moving. The question is - **is it critical for us to know the `patch` version of underlying native SDK?** This however, is a separate discussion - I'll bring it to Slack.